apps/web/src/common/modules/navigations/top-bar/modules/top-bar-search/helper.ts

```diff
@@ -14,3 +23,34 @@ export const createSearchRecent = async (type: SuggestionType, id: string, works
         ErrorHandler.handleError(e);
     }
 };
+
+export const topBarSearchReferenceRouter = (type: Exclude<SearchTab, 'service'>, resourceId: string, workspaceId: string, cloudServiceTypeMap?:CloudServiceTypeReferenceItem):RawLocation => {
+    const { name, data } = cloudServiceTypeMap || {};
+    switch (type) {
+    case 'serviceAccount':
```

Comment:
Suggestion: How about managing and preventing human errors by defining them as constants?

Reply:
Thank you for your confirmation. I will reflect this in the next PR.